MapboxNavigation/Resources/ru.lproj/Abbreviations.plist

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>abbreviations</key>
+	<dict>
+		<key>apartments</key>
+		<string>apts</string>
```

Comment:
The “localizableabbreviations” resource hasn’t been translated into Russian, so we should omit this plist.